operator/internal/manifests/openshift/configure.go

```diff
@@ -212,5 +213,48 @@ func ConfigureOptions(configOpt *config.Options) error {
 		}
 	}
 
+	// Check user workload is enbaled.
+	if !uwam {
+		return nil
+	}
+
+	// Configure user-workload alertmanager when.
+	if len(configOpt.Overrides) == 0 {
+		configOpt.Overrides = map[string]config.LokiOverrides{}
+	}
+
+	lokiOverrides, ok := configOpt.Overrides[tenantApplication]
+	if !ok {
+		lokiOverrides = config.LokiOverrides{
+			Ruler: config.RulerOverrides{
+				AlertManager: &config.AlertManagerConfig{},
+			},
+		}
+
+		configOpt.Overrides[tenantApplication] = lokiOverrides
+		amOverride := &config.AlertManagerConfig{
+			Hosts:           fmt.Sprintf("https://_web._tcp.%s.%s.svc", MonitoringSVCOperated, MonitoringUserwWrkloadNS),
```

Comment:
```suggestion
			Hosts:           fmt.Sprintf("https://_web._tcp.%s.%s.svc", MonitoringSVCUserWorkload, MonitoringUserwWrkloadNS),
```

Reply:
On what basis are you suggesting this? From what I understand, we should be connecting to the operated SVC

Reply:
Oh yes you are right! I was misguided by the server-name we set on the AM client tls config.